Lib/statistics.py

```diff
@@ -111,7 +111,6 @@
 from math import hypot, sqrt, fabs, exp, erf, tau, log, fsum
 from operator import itemgetter
 from collections import Counter
-
```

Comment:
Please add back the spurious whitespace change.

Reply:
Resolved!